docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md

```diff
@@ -42,7 +42,10 @@ documentation.
 
 - Each of your Edge hosts must have at least 4 CPUs and 8 GB of RAM.
 
-- At least 160 GB of persistent storage. The actual amount of storage required depends on the size of your images.
+  - For single-node clusters with Harbor, the Edge host needs at least 6 CPUs and 12 GB of RAM.
```

Comment:
Lenny is this one worker node one control plane node setup? Or is this something different?

Reply:
It's one node that handles both control plane and worker capabilities